### PR TITLE
pythonPackages.ewmh: init at 0.1.6

### PR DESCRIPTION
**Pull request by bennyandresen**
###### Motivation for this change
Include pyewmh library.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/ewmh/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, python-xlib }:
+
+buildPythonPackage rec {
+  pname = "ewmh";
+  version = "0.1.6";
+  name = "${pname}-${version}";
```
Comment (FRidh) [inline pkgs/development/python-modules/ewmh/default.nix]: The `name` attribute is added by `buildPython*` and should therefore be removed.

File: pkgs/development/python-modules/ewmh/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, python-xlib }:
```
Comment (FRidh) [inline pkgs/development/python-modules/ewmh/default.nix]: xlib

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -233,6 +233,10 @@ in {
 
   email_validator = callPackage ../development/python-modules/email-validator { };
 
+  ewmh = callPackage ../development/python-modules/ewmh {
+    python-xlib = self.xlib;
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: drop this

**Review by FRidh — CHANGES_REQUESTED** [review]
Thanks. Some small changes are needed.

**Comment by FRidh** [comment]
@GrahamcOfBorg build python3.pkgs.ewmh

**Comment by FRidh** [comment]
This is an application and should therefore be called directly from `all-packages.nix`, and not via `python-packages.nix`.

**Comment by FRidh** [comment]
```
Ran 0 tests in 0.000s
```
No tests are found. Either the test runner cannot find tests and needs to be patched, or there simply are no tests in which case the tests need to be disabled. Do include a comment explaining why the tests are disabled.

**Comment by bandresen** [comment]
Thanks for the review. I've updated it with the necessary steps except for this:
> This is an application and should therefore be called directly from all-packages.nix, and not via python-packages.nix.

It's a library:
```
/nix/store/5w6by7hwyyck70bcn2qnwrwp9qgpjpfd-python2.7-ewmh-0.1.6
├── lib
│  └── python2.7
│     └── site-packages
│        ├── ewmh
│        │  ├── __init__.py
│        │  ├── __init__.pyc
│        │  ├── ewmh.py
│        │  └── ewmh.pyc
│        └── ewmh-0.1.6.dist-info
│           ├── DESCRIPTION.rst
│           ├── INSTALLER
│           ├── METADATA
│           ├── metadata.json
│           ├── RECORD
│           ├── top_level.txt
│           └── WHEEL
└── nix-support
   └── propagated-build-inputs
```

Could you review it again? @FRidh

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.ewmh python3.pkgs.ewmh

**Review by dotlambda — APPROVED** [review]
It is indeed a library: http://ewmh.readthedocs.io/en/latest/ewmh.html

**Pull request merged**